setup/Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM python:3-alpine
+ARG DISTRO=alpine:3.8
+FROM $DISTRO
 
 RUN mkdir -p /app
 WORKDIR /app
 
 COPY requirements.txt requirements.txt
-RUN apk add --no-cache curl \
- && pip install -r requirements.txt
+RUN apk add --no-cache curl python3 py3-pip
+RUN pip3 install -r requirements.txt
 
```

Comment:
Keep these actions in a single `RUN` statement, in order to avoid additional layers.